test-packages/macro-tests/package.json

```diff
@@ -48,13 +50,13 @@
     "eslint-plugin-node": "^9.0.1",
     "loader.js": "^4.7.0",
     "macro-sample-addon": "0.0.0",
-    "@embroider/funky-sample-addon": "0.0.0",
     "qunit-dom": "^0.8.4"
   },
   "engines": {
     "node": "8.* || >= 10.*"
   },
   "dependencies": {
+    "ember-classic-decorator": "^0.1.3",
```

Comment:
Please stop adding things to the macro-tests app. It exists to test the macro system.  It will be very hard to maintain if it continues to gather a grab bag of behaviors.

Reply:
@ef4 ok, I'll keep it around as I debug the problem and then remove it.